static/js/containers/ProfilePage_test.js

```diff
@@ -127,35 +129,59 @@ describe("ProfilePage", function() {
         TestUtils.Simulate.change(toggle[0]);
         activeDialog('employment-dialog-wrapper');
       };
-      setStep(EMPLOYMENT_STEP);
-      return renderComponent('/profile', successActions).then(dialogTest);
+      return renderComponent('/profile/professional', SUCCESS_ACTIONS).then(dialogTest);
+    });
+  });
+
+  describe('profile completeness', () => {
+    let checkStep = () => helper.store.getState().ui.profileStep;
+
```

Comment:
Yup will do, also just caught that `checkStep` is now defined twice in this file and is also a bit of a misnomer.